WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
-  s.version       = "1.24.1"
+  s.version       = "1.24.thuy"
```

Comment:
There's no need to update this. It is updated to `1.24.1` in the release branch, so we'll just use that updated version when we release.

Reply:
commit reverted :)